### ISSUE TITLE
Translation with placeholder, 0 as data attribute value

### ISSUE THREAD

**Issue by oisti** [ryandrewjohnson/react-localize-redux#116]
Translation placeholder not repaced if the data attribute value is 0.
```
<Translate id="number" data={{zero: 0, one: 1}}>
        {"Number ${zero}, ${one} "}
</Translate>
```
Result: Number ${zero}, 1

```
"react": "16.4.2",
"react-localize-redux": "3.3.0",
```

**Comment by ryandrewjohnson** [comment]
Fixed in #117 and released in `v3.3.1`.

**Comment by oisti** [comment]
Thank you!
